adagrams/game.py

```diff
@@ -1,13 +1,133 @@
 from random import randint
 
+#adding the letter pool to game.py for reference
+LETTER_POOL = {
+    'A': 9, 
+    'B': 2, 
+    'C': 2, 
+    'D': 4, 
+    'E': 12, 
+    'F': 2, 
+    'G': 3, 
+    'H': 2, 
+    'I': 9, 
+    'J': 1, 
+    'K': 1, 
+    'L': 4, 
+    'M': 2, 
+    'N': 6, 
+    'O': 8, 
+    'P': 2, 
+    'Q': 1, 
+    'R': 6, 
+    'S': 4, 
+    'T': 6, 
+    'U': 4, 
+    'V': 2, 
+    'W': 2, 
+    'X': 1, 
+    'Y': 2, 
+    'Z': 1
+}
+
+#including score counter 
+LETTER_SCORE = {
+    1:['A','E','I','O','U','L','N','R','S','T'],
+    2:['D','G'],
+    3:['B','C','M','P'],
+    4:['F','H','V','W','Y'],
+    5:['K'],
+    8:['J','X'],
+    10:['Q','Z']
+}
+
+#function for converting the letter pool in a copied list for use
+def available_letters():
+
+    available_letters = []
+
+    for letter, quantity in LETTER_POOL.items():
+        available_letters.extend(([letter] * quantity))
+
+    return available_letters
+    
+
 def draw_letters():
-    pass
+
+    letters_list = available_letters()
+    letters = []
+    
+    while len(letters) < 10:
```

Comment:
In Python, we call `10` in this context a "magic number". This means that the number has been hardcoded without much context. If I were a developer unfamiliar with Adagrams, I might not know exactly what that 10 means. We can avoid these magic numbers by making constants with more descriptive names to hold them. In this instance, we know that 10 is the max size of the hand, so we might do something along the lines of: 

```suggestion
    # Global Constant:
    MAX_HAND_LENGTH = 10
    while len(letters) < MAX_HAND_LENGTH:
```